src/posts/lucky_1.0.0-rc1.cr

```diff
@@ -0,0 +1,239 @@
+class Lucky100rc1Release < BasePost
+  title "Lucky reaches the 1.0 ascent with the first release candidate"
```

Comment:
What do you think about this:

```suggestion
  title "Lucky begins the 1.0 ascent with the first release candidate"
```

I think that might flow better than "reaches the ascent"